pkg/registry/deployment.go

```diff
@@ -231,6 +233,23 @@ func GenerateDeployment(cr *registryv1alpha1.DevfileRegistry, scheme *runtime.Sc
 					corev1.ResourceMemory: resource.MustParse("256Mi"),
 				},
 			},
+			Env: []corev1.EnvVar{
+				{
+					Name:  "NEXT_PUBLIC_ANALYTICS_WRITE_KEY",
+					Value: cr.Spec.Telemetry.RegistryViewerWriteKey,
+				},
+				{
+					Name: "DEVFILE_REGISTRIES",
+					Value: fmt.Sprintf(`
+					[
+						{
+							"name": "Community",
```

Comment:
We should substitute the `cr.ObjectMeta.Name` here instead of hardcoding it.  Same comment applies to the DEVFILE_REGISTRIES in the configmap.go file above.